zds/settings.py

```diff
@@ -572,6 +570,6 @@
 
 # Load the production settings, overwrite the existing ones if needed
 try:
-    from settings_prod import *
+    from settings_prod import *  # noqa
```

Comment:
`Flake8 Warning F403 — 'from settings_prod import *' used; unable to detect undefined namesat line 575 col 5`